examples/src/document_loaders/blockchain.ts

```diff
@@ -0,0 +1,84 @@
+import { BlockchainPagesLoader } from 'langchain/document_loaders/web/blockchain';
```

Comment:
Can we call this something different since it uses a third-party? Maybe `SortBlockchainLoader`?